frontend/viewer/src/lib/entry-editor/NewEntryDialog.svelte

```diff
@@ -51,12 +68,19 @@
   }
 </script>
 
-<Dialog bind:open on:close={onClosing} {loading} persistent={loading}>
+<Dialog bind:open on:close={onClosing} {loading} persistent={loading} classes={{backdrop: 'overflow-hidden'}}>
   <div slot="title">New {fieldName({id: 'entry'}, $currentView.i18nKey)}</div>
   <div class="m-6">
-    <EntryEditor bind:entry={entry} modalMode/>
+    <OverrideFields shownFields={['lexemeForm', 'citationForm', 'gloss', 'definition']}>
```

Comment:
This editor will include a "Sense 1" header, right?
I wonder if it would be slightly more aesthetically pleasing to avoid that. E.g. by somehow using a <SenseEditor> explicitly out here?

Reply:
I don't think it would go in the override. But yes that's something we could probably improve, though I'm going to leave it for now